EventKit/eventkit-cloud
# Use S3 for all downloads

## Description of Improvement

This PR consolidates all user facing files on S3 and adds minio to the dev environment for the sake of developing and testing.  This doesn't change any APIs and consolidates signficant amounts of duplicate code while removing fragile and complicated branching. 

## How to test

Bring up the application and create a datapack, ensure that the downloads appear and work as expected.  Update or add a data provider that support data previews and ensure that a thumbnail is generated.  Create an export with that provider and ensure that the previews work and can be displayed on the preview page. 

## Quality Assurance 

The following items have been updated:
 
## Screenshots



:smile:
